Application Actions/Firefox/1.0/Scroll Down.scpt

```diff
@@ -0,0 +1,5 @@
+tell application "System Events" to tell process "Firefox"
```

Comment:
There's no need to include `to tell process "Firefox"` since key codes can only be sent to the active app.
You can remove `to tell process XYZ` from all other scripts as well.